bin/testRelease.sh

```diff
@@ -5,14 +5,14 @@ VERSION=$1
 coursier fetch com.lihaoyi:utest_2.10:$VERSION
 coursier fetch com.lihaoyi:utest_2.11:$VERSION
 coursier fetch com.lihaoyi:utest_2.12:$VERSION
-coursier fetch com.lihaoyi:utest_2.13.0-M3:$VERSION
 coursier fetch com.lihaoyi:utest_2.13.0-M5:$VERSION
+coursier fetch com.lihaoyi:utest_2.13.0-M7:$VERSION
```

Comment:
`s/M7/M5/`?

Reply:
Or just don't add this line

Reply:
Fixed. This script is only used to sanity check a release went through.